tsconfig.json

```diff
@@ -1,18 +1,10 @@
 {
+  "include": ["src"],
   "compilerOptions": {
-    "target": "ES2019",
-    "module": "CommonJS",
-    "moduleResolution": "Node",
     "strict": true,
-    "sourceMap": true,
-    "esModuleInterop": true,
-    "lib": ["ESNext", "DOM"],
-    "declaration": true,
-    "noImplicitOverride": true,
-    "noUncheckedIndexedAccess": true,
-    "outDir": "lib",
-    "rootDir": "src"
-  },
-  "include": ["src/**/*.ts"],
-  "exclude": ["node_modules"]
+    "lib": ["esnext", "dom"],
+    "jsx": "preserve",
+    "moduleResolution": "node",
+    "target": "esnext"
+  }
```

Comment:
I'm not sure why you change that much of the project? Please try to minimize the diff as much as possible.

Reply:
back